src/Symfony/Component/Form/Extension/DataCollector/FormDataExtractor.php

```diff
@@ -199,4 +198,43 @@ private function buildId(FormInterface $form)
 
         return $id;
     }
+
+    /**
+     * Converts the variable into a serializable Data instance.
+     *
+     * @param mixed $var
+     *
+     * @return Data
+     */
+    private function cloneVar($var)
+    {
+        if (null === $this->cloner) {
+            $this->cloner = new VarCloner();
+            $this->cloner->addCasters(array(
+                \Exception::class => function (\Exception $e, array $a, Stub $s) {
+                    if (isset($a[$k = "\0Exception\0previous"])) {
+                        unset($a[$k]);
+                        ++$s->cut;
+                    }
+
+                    return $a;
+                },
+                FormInterface::class => function (FormInterface $f, array $a, Stub $s) {
+                    return array(
+                        Caster::PREFIX_VIRTUAL.'name' => $f->getName(),
+                        Caster::PREFIX_VIRTUAL.'type_class' => get_class($f->getConfig()->getType()->getInnerType()),
```

Comment:
renamed this to type_class to be consistent with https://github.com/nicolas-grekas/symfony/pull/6/files#diff-1b203840c96e2f4a2f0386788e2765d3R53